frontend/src/components/footer.css

```diff
@@ -98,4 +103,19 @@ nav {
 
 .moveConnecting{
    margin-right:35px !important;
+}
+
+.footer-contact {
+  position: absolute;
```

Comment:
As a general rule of thumb, unless it is absolutely vital to do so, almost nothing should be `position: absolute`. What's the rationale here?